.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "blockchain-k-plugin"]
 	path = deps/blockchain-k-plugin
-	url = https://github.com/runtimeverification/blockchain-k-plugin
+	url = https://github.com/runtimeverification/blockchain-k-plugin
```

Comment:
Do you need this change?

Reply:
I don't think we do, but for some reason, I wasn't able to clone the submodules after some modifications to the project. After re-adding the K blockchain submodule, it marked this line as a diff; the reason is unclear to me. I'd keep this as it is, as it doesn't harm the project.